src/sentry/hybridcloud/models/webhookpayload.py

```diff
@@ -46,6 +48,24 @@ class Meta:
         indexes = (
             models.Index(fields=["mailbox_name"]),
             models.Index(fields=["schedule_for"]),
+            models.Index(fields=["provider"], name="webhookpayload_provider_idx"),
+            models.Index(
```

Comment:
@wedamija @markstory I added this index for the `head_of_line` sub-query here: https://github.com/getsentry/sentry/blob/adffd898c80c522d0b70c1f0be35cec0a2bf1fb1/src/sentry/hybridcloud/tasks/deliver_webhooks.py#L82-L89

Reply:
lgtm